gradio/analytics.py

```diff
@@ -227,6 +232,33 @@ def launched_analytics(blocks: gradio.Blocks, data: dict[str, Any]) -> None:
     _do_analytics_request(url=f"{ANALYTICS_URL}gradio-launched-telemetry/", data=data)
 
 
+def custom_component_analytics(
+    command: str,
+    template: str | None,
+    upload_pypi: bool | None,
+    upload_demo: bool | None,
+    upload_source: bool | None,
+    generate_docs: bool | None = None,
+    bump_version: bool | None = None,
+) -> None:
```

Comment:
Do you think it would be worth capturing the arguments being used (as a list) but maybe not the value. Then we could see if people are customising extensively or not?